doc/examples/transform/plot_masked_register_translation.py

```diff
@@ -47,9 +47,9 @@
 # the pixels ourselves
 mask = corrupted_pixels
 
-shift = masked_register_translation(image, offset_image, mask)
+detected_shift = masked_register_translation(image, offset_image, mask)
 
-print("Detected pixel offset (y, x): {}".format(shift))
+print("Detected pixel offset (y, x): {}".format(-detected_shift))
```

Comment:
Isn't it confusing to print the opposite value of the value returned by the function?

Reply:
This point was [discussed](https://github.com/scikit-image/scikit-image/pull/3334#discussion_r231680321) at the end of #3334.

Just like the `register_translation` function, the shift returned by `masked_register_translation` is the shift required to _shift back_. I personally find this confusing to see in the a gallery example (see gallery example for `register_translation` [here](http://scikit-image.org/docs/stable/auto_examples/transform/plot_register_translation.html#sphx-glr-auto-examples-transform-plot-register-translation-py)). 

Maybe, instead of calling it _detected offset_, we should call it _corrective offset_?

Reply:
I'm ambivalent about all this (I can see arguments for either convention), but I would like to ask that `y, x` be changed to `r, c`. See [coordinate conventions](http://scikit-image.org/docs/dev/user_guide/numpy_images.html#coordinate-conventions).

Reply:
@jni good suggestion, I've changed it in latest commit